jmxtrans-output/jmxtrans-output-core/src/test/java/com/googlecode/jmxtrans/model/output/OpenTSDBGenericWriterTests.java

```diff
@@ -67,6 +70,7 @@
 	public void setupTest() {
 		this.mockQuery = Mockito.mock(Query.class);
 		this.mockResult = Mockito.mock(Result.class);
+		this.server = ServerFixtures.dummyServer();
```

Comment:
Using `ServerFixutures` also means that you don't need to keep server as instance variable, you can just inline it everywhere it is used.